### DISCUSSION TITLE
Filter and column dropdown not working

### DISCUSSION THREAD

**Discussion by Zeri-web** [rappasoft/laravel-livewire-tables#942] [Q&A]
I'm new using rappasoft v2, After follow the installation guide and creating filters, my filter is working, but cannot click and it display like image below. Anyone who experienced this before? 
![Capture](https://user-images.githubusercontent.com/85867903/195495459-5151dfab-e56e-404d-92e4-bbefa1d780c9.PNG)

EDIT:
The problem is solved, I'm put this <script defer src="https://unpkg.com/alpinejs@3.x.x/dist/cdn.min.js"></script> in my master layout

**Comment by AbhinavNmdo** [comment]
Can you please give me your code because I didn't get the dropdown .

![image](https://user-images.githubusercontent.com/86583211/202354277-eecd0902-e9d6-4155-964f-c59020d6ece1.png)

**Reply by Zeri-web** [reply]
put this code inside master blade

 <script defer src="https://unpkg.com/alpinejs@3.x.x/dist/cdn.min.js"></script>

![image](https://user-images.githubusercontent.com/85867903/204231841-172bc3c5-512f-4d85-9456-fcef704484f8.png)
